### PR TITLE
Fix golint failures in controller,driver and identityserver.

### PR DESCRIPTION
**Pull request by humblec**
Fixes #https://github.com/gluster/gluster-csi-driver/issues/55

Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

**Describe what this PR does**
Provide some context for the reviewer

**Is there anything that requires special attention?**
Do you have any questions? Did you do something clever?

**Related issues:**
Mention any github issues relevant to this PR

### CODE REVIEW

**Comment by humblec** [comment]
@Madhu-1 @JohnStrunk PTAL.

File: pkg/glusterfs/controllerserver.go [hunk 1]
```diff
@@ -374,17 +375,17 @@ func (cs *ControllerServer) ControllerGetCapabilities(ctx context.Context, req *
 	return resp, nil
 }
 
-//CreateSnapshot
+//CreateSnapshot create snapshot of an existing PV
```
[the remark this answers is not in the document]
Comment (humblec) [reply pkg/glusterfs/controllerserver.go]: @Madhu-1 PVC:PV is 1:1 , so any will do.

File: pkg/glusterfs/identityserver.go [hunk 2]
```diff
@@ -7,6 +7,7 @@ import (
 	"github.com/golang/glog"
 )
 
+//IdentityServer is the struct with the reference to glusterfs CSI driver.
 type IdentityServer struct {
```
Comment (Madhu-1) [inline pkg/glusterfs/identityserver.go]: >//NodeServer of Glusterfs CSI driver with supported methods of CSI node server spec.

@humblec can you add the same comment here also as you added for `NodeServer` struct.

File: pkg/glusterfs/controllerserver.go [hunk 3]
```diff
@@ -24,6 +24,7 @@ const (
 
 var errVolumeNotFound = errors.New("volume not found")
 
+//ControllerServer is the struct with the reference to glusterfs CSI driver.
```
Comment (Madhu-1) [inline pkg/glusterfs/controllerserver.go]: same here as well for `ControllerServer` struct

File: pkg/glusterfs/identityserver.go (hunk 2 above)
Comment (humblec) [reply pkg/glusterfs/identityserver.go]: @madhu-1 made it consistent

Comment (Madhu-1) [reply pkg/glusterfs/identityserver.go]: @humblec I prefer your old comment. it looks more meaningful as per implementation, please let me know your thoughts on this

**Comment by JohnStrunk** [comment]
I'm not interested in being :policeman: for this, but these comments serve as the main developer documentation for the driver. @humblec and @Madhu-1 you two can decide where you want to go w/ this. I think it would be great if we had a good, professional set of docs:
https://godoc.org/github.com/gluster/gluster-csi-driver/pkg/glusterfs

**Comment by humblec** [comment]
@JohnStrunk Nice comment :P,  regarding the doc, yes we need to come up with good set of docs. May be we can improve on the way. My main aim was to get rid of the golint issues. So if @Madhu-1 agree, we can get this rolling and keep improving our source doc.

**Comment by Madhu-1** [comment]
@humblec if we update the `NodeServer` `ControllerServer` and `IdentityServer` struct comments we are good to go.

I like this comment on `NodeServer`, can we have the same comment for other structs also?
```
//NodeServer of Glusterfs CSI driver with supported methods of CSI node server spec.
```

**Comment by humblec** [comment]
@Madhu-1 ok, addressing it.

**Comment by humblec** [comment]
@Madhu-1 Done. PTAL

**Comment by Madhu-1** [comment]
Sorry I didn't notice earlier I think we need to have a space between `//` and comment 
`//NodeServer` to `// NodeServer`

**Comment by humblec** [comment]
@Madhu-1 The space after `\\` has been removed from all the files. I dont know having a space or without space is better. But its consistent now.

**Comment by Madhu-1** [comment]
@humblec i have seen some comments  [here](https://github.com/golang/go/wiki/Comments#comments)
i see space after `//` and check this [blog on golang](https://dmitri.shuralyov.com/idiomatic-go#comments-for-humans-always-have-a-single-space-after-the-slashes) they also suggest the same

**Comment by humblec** [comment]
@Madhu-1 lets keep the space everywhere then. I am resending.

**Comment by humblec** [comment]
@Madhu-1 done

**Pull request merged**